net/local/local_fifo.c

```diff
@@ -603,6 +657,17 @@ int local_open_receiver(FAR struct local_conn_s *conn, bool nonblock)
       /* Policy: Free FIFO resources when the buffer is empty. */
 
       ret = local_set_policy(&conn->lc_infile, 1);
+
+      if (ret == 0)
```

Comment:
only nonblock need set threshold?

Reply:
The key thought is:
POLLIN/ POLLOUT shouldn't happen if only preamble can be buffered, whether block or nonblock.

Though only nonblock will lead to a much worse result: packet receiving failure. I don't think we should distinguish between block and non-block.

Reply:
Ok.